test ppt/unit-tests/extT.cc fails due to missing return code

When doing a make -j check, I see this warning:

```
extT.cc: In member function ‘int extT::check_extensions(int, std::map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >&, std::map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> >&)’:
extT.cc:106:5: warning: no return statement in function returning non-void [-Wreturn-type]
  106 |     }
      |     ^
```

And the test fails.

Looking in the code, I see that the int function is returning without a value. I added a "return 0" and now the test works.

--- a/ppt/unit-tests/extT.cc
+++ b/ppt/unit-tests/extT.cc
@@ -103,6 +103,7 @@ CPPUNIT_TEST_SUITE( extT );
             cout << "var " << var << " = " << (*f).second << ", should be " << val << endl;
             CPPUNIT_ASSERT( (*f).second == val );
         }
+	return 0;
     }
 
     void do_try_list()

